src/wrapper-utils.js

```diff
@@ -0,0 +1,131 @@
+/* global cloneInto, exportFunction, mozProxies */
+// Tests don't define this variable so fallback to behave like chrome
+const hasMozProxies = typeof mozProxies !== 'undefined' ? mozProxies : false
+const globalObj = typeof window === 'undefined' ? globalThis : window
+const getOwnPropertyDescriptor = Object.getOwnPropertyDescriptor
+const functionToString = Function.prototype.toString
+const objectKeys = Object.keys
```

Comment:
These probably belong more in src/captured-globals.js which just landed (I assume you're capturing them to prevent extension).

You might need to capture the descriptors there if you **need** them. However I think it's OK to not be perfect if it's only ever internally used like we are here.

We'd just have:
```suggestion
import { Object, Function } from './captured-globals.js'
```

Reply:
As discussed, the current implementation in captured-globals has some limitations, so we'll address it in a separate task.